tool/gravity/cli/commands.go

```diff
@@ -654,6 +654,8 @@ type UpdateUploadCmd struct {
 	*kingpin.CmdClause
 	// OpsCenterURL is cluster URL
 	OpsCenterURL *string
+	// Force allows to override certain preconditions.
```

Comment:
The uncertainty is beginning to make me feel uneasy. Maybe we should break the preconditions down to a list and provide clear override rules for each or just evaluate whether a generic override makes sense for all?

Reply:
Yeah good call. Renamed this to `--skip-version-check`.